src/main/java/no/sikt/nva/scrapers/DublinCoreScraper.java

```diff
@@ -681,25 +681,34 @@ private static PartOfSeries extractPartOfSeries(DublinCore dublinCore) {
                                      .filter(DcValue::isPartOfSeries)
                                      .map(DcValue::scrapeValueAndSetToScraped)
                                      .collect(Collectors.toList());
-       return convertListToPartOfSeries(partOfSeriesValues);
+       return convertListToPartOfSeries(partOfSeriesValues, dublinCore);
     }
 
-    private static PartOfSeries convertListToPartOfSeries(List<String> list) {
+    private static PartOfSeries convertListToPartOfSeries(List<String> list, DublinCore dublinCore) {
         if (isSingleton(list)) {
             var parts = Arrays.stream(list.get(0).split(";"))
                             .map(String::trim)
                             .filter(value -> !value.isEmpty())
                             .collect(Collectors.toList());
-            return constructPartOfSeries(parts);
+            return constructPartOfSeries(parts, dublinCore);
         } else {
-            return constructPartOfSeries(list);
+            return constructPartOfSeries(list, dublinCore);
         }
     }
 
-    private static PartOfSeries constructPartOfSeries(List<String> list) {
+    private static PartOfSeries constructPartOfSeries(List<String> list, DublinCore dublinCore) {
         var name = attempt(() -> list.get(0)).orElse(failure -> null);
-        var number = attempt(() -> list.get(1)).orElse(failure -> null);
-        return new PartOfSeries(name, number);
+        var number = Optional.ofNullable(attempt(() -> list.get(1)).orElse(failure -> null))
+                         .or(() -> Optional.ofNullable(extractPartOf(dublinCore)))
+                         .orElse(null);
+        return isSeriesName(name)
+                   ? new PartOfSeries(name, number)
+                   : new PartOfSeries(number, name);
```

Comment:
I think this is its own method since what we're doing here is a bit odd. Maybe `extractSeries`